starknet_py/net/schemas/gateway.py

```diff
@@ -108,6 +108,7 @@ def make_dataclass(self, data, **kwargs) -> InvokeTransaction:
 
 class DeployTransactionSchema(TransactionSchema):
     contract_address = Felt(data_key="contract_address", required=True)
+    contract_address_salt = Felt(data_key="contract_address_salt", required=True)
```

Comment:
Is this connected to RPC?
If not, could we have it in another PR for #1119 ? 🙏

Reply:
Indirectly, yes, this field wasn't included neither in DeployTransaction, nor in schemas, but both clients return it. If I added it to only one schema, then it would have to be marked Optional only to make it not optional in some other PR.